src/main/java/com/uid2/client/IdentityInput.java

```diff
@@ -0,0 +1,93 @@
+package com.uid2.client;
+
+import com.google.gson.JsonObject;
+
+import java.security.MessageDigest;
+
+public class IdentityInput {
+    static public IdentityInput fromEmail(String email) {
+        return new IdentityInput(IdentityType.Email, email, true);
+    }
+
+    static public IdentityInput fromPhone(String phone) {
+        return new IdentityInput(IdentityType.Phone, phone, true);
+    }
+
+    public IdentityInput withTcfConsentString(String tcf) {
```

Comment:
Check what the correct term for this string is and let's make sure that variables, data members, method names are consistent.

Reply:
Sure. Important to note this will mean the SDK uses a different term to the API https://github.com/UnifiedID2/EUID-docs/blob/main/api/v2/endpoints/post-token-generate.md (is it too late to change API?)